component.css

```diff
@@ -0,0 +1,549 @@
+.image-circle-small{
+    width: 250px;
+    height: 250px;
+    border-radius: 50%;
+}
+.image-circle-medium{
+    width: 400px;
+    height: 400px;
+    border-radius: 50%;
+}
+.image-circle-large{
+    width: 500px;
+    height: 500px;
+    border-radius: 50%;
+}
+.image-responsive{
+    height: auto;
+}
+                                                /* List  */
+.list{
+    border: 1px solid rgb(192, 191, 191);
+    border-radius: 10px;
+    width: fit-content;
+}
+.list-items{
+    padding: 10px;
+    list-style: none;
+    text-align: center;
+    border-bottom: 1px solid rgb(192, 191, 191);
+}
+.top-item{
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
+}
+.bottom-item{
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+    border-bottom:none;
+}
+.active{
+    color: white;
+    background-color: var(--primary);
+}
+.notification-item{
+    padding: 10px;
+    list-style: none;
+    text-align: center;
+    border-bottom: 1px solid rgb(192, 191, 191);
+    display: flex;
+    align-items: center;
+}
+                                             /* Input  */
+
+.inputbox{
+    box-sizing: border-box;
+    width: 250px;
+    height: 40px;
+    margin: 10px;
+}
+.input-field{
+    position: relative;
+    width: 100%;
+    height: 100%;
+    border: none;
+    outline: none;
+    font-size: 18px;
+    border-bottom: 2px solid black;
+}
+.input-field:focus,.input-field-valid:focus{
+    border-bottom-color:var(--primary);
+}
+.input-heading{
+    position: relative;
+    transition: 0.2s;
+    top: -35px;
+    pointer-events: none;
+}
+.input-field:focus + .input-heading,.input-field:valid + .input-heading{
+    top: -59px;
+    font-size: 12px;
+    color: grey;
+}
+
+.inputbox-valid{
+    width: 90%;
+    margin: auto;
+}
+.input-field-valid{
+    position: relative;
+    width: 100%;
+    height: 100%;
+    border: none;
+    outline: none;
+    font-size: 18px;
+    border-bottom: 2px solid black;
+    margin: 10px 0;
+}
+.valid-status{
+    color:var(--grey-text);
+    font-size: 14px;
+}
+
+                                       /* Modal */
+.modal-container{
+    display: none;
+    position: fixed;
+    overflow: auto;
+    left: 40%;
+    top: 10px;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    opacity: 1;
+}
+.modal{
+    border: 1px solid black;
+    width: 300px;
+    background-color: white;
+}
+.modal-header{
+   text-align: center;
+   padding: 10px 0;
+}
+.modal-body{
+    border-top: 1px solid gray;
+    padding: 10px;
+    border-bottom: 1px solid gray;
+}
+.modal-button{
+    margin: 10px;
+}
+                                       /* Navbar */
+.nav-primary{
+    background-color: var(--primary);
+}
+.nav-secondary{
+    background-color:var(--secondary);
+}
+.nav-box{
+    display: flex;
+    width: fit-content;
+    flex-wrap: wrap;
+    margin: auto;
+}
+.navelem{
+    list-style: none;
+    padding: 10px;
+}
+.navtext-primary,.navtext-secondary{
```

Comment:
Is comma a typo here? never seen its use.

Reply:
Comma is used if you want multiple classes to have same properties